003_four_in_line_oop/src/domain/Plosca.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Edin on 23. 03. 2022.
+//
+
+#ifndef INSTRUKCIJE_D001_PLOSCA_H
+#define INSTRUKCIJE_D001_PLOSCA_H
+
+#include "Zeton.h"
+
+#define PLOSCA_ST_ZETONOV 42
+#define PLOSCA_ST_VRSTIC 6
+#define PLOSCA_ST_STOLPCEV 7
+
+typedef struct Plosca {
+    int height;
+    int width;
+    Zeton zetoni[PLOSCA_ST_ZETONOV];
+    bool active;
+    int frames; // kdo bo na vrsti (omejen z PLOSCA_ST_ZETONOV)
+    int memo_tabela[PLOSCA_ST_VRSTIC][PLOSCA_ST_STOLPCEV];
+    int stolpci[PLOSCA_ST_STOLPCEV][PLOSCA_ST_VRSTIC];
+    int counterjiStolpcev[PLOSCA_ST_STOLPCEV];
```

Comment:
Plošča naj ima samo
    int height;
    int width;
    Zeton zetoni[PLOSCA_ST_ZETONOV];
    bool active;

Reply:
To pomeni da bo potrebno logiko v funkcijah kjer uporabljaš te stvari na novo napisati.